tools/dia_dll.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# Copyright (c) 2012 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+This script is based on chromium/chromium/master/tools/clang/scripts/update.py.
+
+It is used on Windows platforms to copy the correct msdia*.dll to the
+clang folder, as a "gclient hook".
+"""
+
+import os
+import shutil
+import stat
+import sys
+
+
+# Path constants. (All of these should be absolute paths.)
+THIS_DIR = os.path.abspath(os.path.dirname(__file__))
+LLVM_BUILD_DIR = os.path.abspath(os.path.join(THIS_DIR, '..', '..', 'third_party',
+                                              'llvm-build', 'Release+Asserts'))
+
+
+def GetDiaDll():
```

Comment:
What is this DLL for?

Reply:
This is what Chromium does upstream to build with.  Without this, the build fails because it can't find expected symbols.  Don't really have more details beyond that.